content/source/docs/cloud/registry/using.html.md

```diff
@@ -36,35 +36,23 @@ The module page appears, containing a list of all available modules. You can fil
 
 ### Shared Modules - Terraform Enterprise
 
-On Terraform Enterprise, your [module sharing](/docs/enterprise/admin/module-sharing.html) configuration may grant you access to modules outside your organization. Modules that are shared with your current organization have a "Shared" badge.
+On Terraform Enterprise, your [module sharing](/docs/enterprise/admin/module-sharing.html) configuration may grant you access to modules outside your organization. Modules that are shared with your current organization have a "Shared" badge (below). Modules in your current organization that are shared with other organizations have a badge that says "Sharing".
 
 ![Terraform Enterprise screenshot: shared module](./images/using-modules-list-shared.png)
 
-Modules in your current organization that are shared with other organizations have a badge that says "Sharing".
-
-![Terraform Enterprise screenshot: sharing module](./images/using-modules-list-sharing.png)
 
 ### Viewing Module Details and Versions
 
-Click a module's "Details" button to view its details page. Use the "Versions" dropdown in the upper right to switch between the available versions, and use the Readme/Inputs/Outputs/Dependencies/Resources tabs to view detailed documentation and information about a version.
```

Comment:
was this not accurate? I still see the detail button on the screen shot?

Reply:
Yeah when I go into TFC and I go to Registry and want to look at the available modules, there's no details button when I mouse over. I just mouse over and have to click it and then it goes to the details page. Thank you for flagging that on the shared TFE image!  I got in touch with the TFE team and they gave me a new image from a recent instance that doesn't show the details button (they must have taken that out). I pushed up the new image, so we should be all set!